cove_360/lib/threesixtygiving.py

```diff
@@ -505,7 +505,7 @@ class LooksLikeEmail(AdditionalTest):
 
     check_text = {
         "heading": "text that looks like an email address",
-        "message": "This may indicate that the data contains personal data, use and distribution of which is restricted by the Data Protection Act. You should ensure that any personal data is included with the knowledge and consent of the person to whom it refers."
+        "message": "An @ symbol has been found in your data. This may indicate that the data contains an email address, which can constitute personal data. The use and distribution of personal data  is restricted by the Data Protection Act. You should ensure that any personal data is only included with the knowledge and consent of the person to whom it refers."
```

Comment:
This isn't quite true, because "pets@home" wouldn't trigger this check, nor would a Twitter handle. It looks for a string with some text, then an @, then some more text that must include at least one dot. If this check has triggered, it almost certainly is because of an email address.